testproject/Assets/Tests/Runtime/MultiprocessRuntime/readme.md

```diff
@@ -196,10 +196,42 @@ Note that performance tests should be run from external processes (not from edit
 
 ## How it's done
 ### Multiple processes orchestration
-The test runner executes the main node's tests. The tests are in charge of launching their needed workers.
+
+Test code and host code execute in the same process: When writing play mode tests, Unity will start a unity environment for that test, including game loop, scene, object hierarchy, all that fun stuff. This means the test itself has access to everything other unity scripts would have access to. At test startup, the test will ask unity to switch scene to MultiprocessTestScene containing a GameObject already placed in that scene called TestCoordinator . The test will callStartHost that will listen for connections, still all in the same process.
+Once that’s done, that same test will then spawn new client processes. These clients will also start with that MultiprocessTestScene  loaded at startup, also containing a TestCoordinator . These client side TestCoordinators will detect they are clients (using command line arg) and instead of calling StartHost  will call StartClient . This is where new code to specify the IP to connect to would stand (the lines I sent you).
+A good way I could have clarified that code is to separate TestCoordinator into TestCoordinatorClient and TestCoordinatorServer thinking of it. Right now TestCoordinator code does both.
+Once the connection is established (tests yield wait for connection in  Setup code), then the tests can start sending RPCs to each other.
+Like Jeff said, the test (that’s server side) will call multiple TriggerActionIdClientRpc . This will trigger these actions on all clients. The clients execute their test code, then answer back with ClientFinishedServerRpc.
```

Comment:
As a user, I don't know who the @jeffreyrainy is :)

Reply:
That's why I should remove my ctrl, c and v keys from my keyboard
fixed